Art/vfx meteor impact explosion

vylam09122001 opened the pull request.

skinny-k reviewed (changes requested):
Way too many changes to files that should not be changed. Discard unnecessary changes.

skinny-k committed:
Merge branch 'dev' into art/vfx_meteor-impact-explosion

skinny-k committed:
PR Fix

Did some Git wizardry to hopefully fix the oversized PR and prevent unexpected changes.

skinny-k reviewed (approved):
Unintended file changes have been repaired.
